platform/libs.jsvg/nbproject/project.properties

```diff
@@ -15,12 +15,12 @@
 # specific language governing permissions and limitations
 # under the License.
 
-file.reference.jsvg-1.6.1.jar=external/jsvg-1.6.1.jar
-release.external/jsvg-1.6.1.jar=modules/ext/jsvg-1.6.1.jar
+file.reference.jsvg-1.7.1.jar=external/jsvg-1.7.1.jar
+release.external/jsvg-1.7.1.jar=modules/ext/jsvg-1.7.1.jar
 
 is.autoload=true
 javac.source=1.8
 
 nbm.homepage=https://github.com/weisJ/jsvg
 sigtest.gen.fail.on.error=false
-spec.version.base=1.23.0
+spec.version.base=2.0.0
```

Comment:
Let's remove this, but add `spec.version.base.fatal.warning=false`.  Everything else could go in manifest, like [FlatLaf](https://github.com/apache/netbeans/blob/master/platform/libs.flatlaf/manifest.mf).